### PR TITLE
#230 & #231

### PR DESCRIPTION
**Pull request by Clemcarle**
Issues #230 and #231 (paper-rock-scissors dynamics) have been addressed.

### CODE REVIEW

File: color.cpp [hunk 1]
```diff
@@ -87,6 +87,13 @@ std::string to_str(const color& in_color) noexcept
   return msg;
 }
 
+bool is_first_color_winner(const color& c1, const color& c2)
+{
+    double hue1 = calc_hue(c1);
```
Comment (ollyturner) [inline color.cpp:92]: This is a bit of nitpicking, yet here goes: this should be const double, as hue_1 is not changed. Still, the tests pass, so I will make that into a newbie Issue 🙂

File: color.cpp [hunk 2]
```diff
@@ -87,6 +87,13 @@ std::string to_str(const color& in_color) noexcept
   return msg;
 }
 
+bool is_first_color_winner(const color& c1, const color& c2)
+{
+    double hue1 = calc_hue(c1);
+    double hue2 = calc_hue(c2);
+
+    return (hue1 == 0.0 && hue2 == 120.0) || (hue1 == 120.0 && hue2 == 240.0) || (hue1 == 240.0 && hue2 == 0.0);
```
Comment (ollyturner) [inline color.cpp:95]: This was all that was needed to pass the test, well done.
Sidenote : there will be more tests 😸.

File: color.cpp [hunk 3]
```diff
@@ -173,7 +180,7 @@ void test_color()
 #endif // FIX_ISSUE_229
 
 
-  //#define FIX_ISSUE_230
+#define FIX_ISSUE_230
```
Comment (ollyturner) [inline color.cpp:183]: This line can be romoved along with the other #ifstatements. @richelbilderbeek will make this cleaning up a newbie issue. 👍 :

**Review by ollyturner — APPROVED** [review]
Well done 🎉 You did an excellent job. Take a look at the comments, but no drastic changes needed.

**Comment by richelbilderbeek** [comment]
Well done @kausmees and @Clemcarle ! Me and @ollyturner checked your handiwork!

**Pull request merged**